src/main.rs

```diff
@@ -236,85 +238,234 @@ fn get_setup(args: &[String]) -> Setup {
     const VOLUME_RANGE: &str = "volume-range";
     const ZEROCONF_PORT: &str = "zeroconf-port";
 
+    // Mostly arbitrary.
+    const AUTOPLAY_SHORT: &str = "A";
+    const AP_PORT_SHORT: &str = "a";
+    const BACKEND_SHORT: &str = "B";
+    const BITRATE_SHORT: &str = "b";
+    const SYSTEM_CACHE_SHORT: &str = "C";
+    const CACHE_SHORT: &str = "c";
+    const DITHER_SHORT: &str = "D";
+    const DEVICE_SHORT: &str = "d";
+    const VOLUME_CTRL_SHORT: &str = "E";
+    const VOLUME_RANGE_SHORT: &str = "e";
+    const DEVICE_TYPE_SHORT: &str = "F";
+    const FORMAT_SHORT: &str = "f";
+    const DISABLE_AUDIO_CACHE_SHORT: &str = "G";
+    const DISABLE_GAPLESS_SHORT: &str = "g";
+    const DISABLE_CREDENTIAL_CACHE_SHORT: &str = "H";
+    const HELP_SHORT: &str = "h";
+    const CACHE_SIZE_LIMIT_SHORT: &str = "M";
+    const MIXER_TYPE_SHORT: &str = "m";
+    const ENABLE_VOLUME_NORMALISATION_SHORT: &str = "N";
+    const NAME_SHORT: &str = "n";
+    const DISABLE_DISCOVERY_SHORT: &str = "O";
+    const ONEVENT_SHORT: &str = "o";
+    const PASSTHROUGH_SHORT: &str = "P";
+    const PASSWORD_SHORT: &str = "p";
+    const EMIT_SINK_EVENTS_SHORT: &str = "Q";
+    const QUIET_SHORT: &str = "q";
+    const INITIAL_VOLUME_SHORT: &str = "R";
+    const ALSA_MIXER_DEVICE_SHORT: &str = "S";
+    const ALSA_MIXER_INDEX_SHORT: &str = "s";
+    const ALSA_MIXER_CONTROL_SHORT: &str = "T";
+    const NORMALISATION_ATTACK_SHORT: &str = "U";
+    const USERNAME_SHORT: &str = "u";
+    const VERSION_SHORT: &str = "V";
+    const VERBOSE_SHORT: &str = "v";
+    const NORMALISATION_GAIN_TYPE_SHORT: &str = "W";
+    const NORMALISATION_KNEE_SHORT: &str = "w";
+    const NORMALISATION_METHOD_SHORT: &str = "X";
+    const PROXY_SHORT: &str = "x";
+    const NORMALISATION_PREGAIN_SHORT: &str = "Y";
+    const NORMALISATION_RELEASE_SHORT: &str = "y";
+    const NORMALISATION_THRESHOLD_SHORT: &str = "Z";
+    const ZEROCONF_PORT_SHORT: &str = "z";
+
+    // Options that have different desc's
+    // depending on what backends were enabled at build time.
+    #[cfg(feature = "alsa-backend")]
+    const MIXER_TYPE_DESC: &str = "Mixer to use {alsa|softvol}. Defaults to softvol.";
+    #[cfg(not(feature = "alsa-backend"))]
+    const MIXER_TYPE_DESC: &str = "Not supported by the current audio backend(s).";
```

Comment:
Do you think "current" is concise enough? I'm wondering if it should be "compiled" or "included" or something like that.

Reply:
I like included it's less technical sounding than compiled and I think easier for average Joe to understand. I can change it.

Reply:
Done.